backend/src/main/kotlin/app/ehrenamtskarte/backend/application/webservice/schema/create/OrganizationContact.kt

```diff
@@ -0,0 +1,29 @@
+package app.ehrenamtskarte.backend.application.webservice.schema.create
+
+import app.ehrenamtskarte.backend.application.webservice.schema.view.JsonField
+import app.ehrenamtskarte.backend.application.webservice.schema.view.Type
+import app.ehrenamtskarte.backend.application.webservice.utils.JsonFieldSerializable
+
+data class OrganizationContact(
+    val name: String,
+    val telephone: String,
+    val email: String,
+    val hasGivenPermission: Boolean
```

Comment:
I haven't seen a hasGivenPermission checkbox anywhere in the applications forms, I don't think it's necessary.

Reply:
I found this field in the mockups for the NRW app and I think it makes sense to include it for data-privacy-legal reasons.
Sowas wie "Die Kontaktperson hat der Datenweitergabe zugestimmt." oder ähnlich... Kannst ja vlt auch nochmal kurz in die Mockups von NRW schauen :)